Java/PascalsTriangle118.java

```diff
@@ -0,0 +1,29 @@
+import java.util.ArrayList;
+import java.util.List;
+
+class Solution {
+    public List<List<Integer>> generate(int numRows) {
+
+        List<List<Integer>> triangle = new ArrayList<>();
+
+        if(numRows ==0) return triangle;
+
+        List<Integer> first_row = new ArrayList<>();
+        first_row.add(1);
+        triangle.add(first_row);
+
+        for(int i=1; i<numRows; i++){
+            List<Integer> prev_row = triangle.get(i-1);
+            List<Integer> row = new ArrayList<>();
+            row.add(1);
```

Comment:
O(n2) Solution is not acceptable

Reply:
this is the best time complexity it shows faster than 100% of submissions with 0ms runtime when submitted, and the space complexity is O(N) i will change that now